apprentice_teacher.rb

```diff
@@ -1,21 +1,14 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
+require_relative 'teacher'
+require_relative 'high_five'
+class ApprenticeTeacher < Teacher
+  include High_five
+  attr_reader :age, :salary, :phase, :target_raise, :rating
```

Comment:
Usually for modules that are going to be included in classes, we use "-able" names, so here it'd be HighFiveable.